collection.go

```diff
@@ -77,6 +77,10 @@ type Collection interface {
 	// Truncate removes all documents from the collection, but leaves the indexes intact.
 	Truncate(ctx context.Context) error
 
+	// Rename renames the collection.
+	// If the collection does not exist, a NotFoundError is returned.
+	Rename(ctx context.Context, options SetRenameOptions) error
```

Comment:
Although passing an options object here is obviously a better solution than passing only a newName, we should stick with existing style. 

Please change it simple `Rename(ctx context.Context, newName string) error`.
See `View` interface which already has Rename function.

Reply:
Thanks for your review. 
I've adapted the changes, now the interface function matches the View interface.

I am having some trouble running my tests locally, all of them return "No endpoints found", is there any guide for setting up the test environment locally?

Reply:
I followed MAINTAINERS.md.

I ran all tests and seem to be passing locally, is it possible to manually trigger the CI or do i need to make another commit?